swiftpkg/tests/swiftpkg_build_files_tests.bzl

```diff
@@ -566,6 +570,11 @@ swift_binary(
         "@rules_swift_package_manager//config_settings/spm/configuration:release": ["-cross-module-optimization"],
         "//conditions:default": [],
     }),
+    features = [
+        "swift.enable_v6",
+        "swift.experimental.BuiltinModule",
+        "swift.upcoming.ExistentialAny",
+    ],
```

Comment:
Should this be done via features or via `copts`? I think features is more readable but requires rules_swift to be updated for new ones where-as `copts` could support things we do not know of yet.

Thoughts @brentleyjones

Reply:
> rules_swift won't need to update for new ones (the way it transforms it is dynamic). Given that, I think it doesn't matter? Features could be better if the copts ever change for how one would enable them, but I doubt that matters.